freeipa/src/main/resources/freeipa-salt/salt/freeipa/backups.sls

```diff
@@ -15,6 +15,10 @@
     - mode: 640
 
 {% if salt['pillar.get']('freeipa:backup:monthly_full_enabled') %}
+'/sbin/anacron -u cron.monthly':
```

Comment:
could you rename this state and move the command into the `name` parameter?
also this would be called every time a highstate is called, so I think you should place a file somewhere so it wouldn't run again

Reply:
It will not be run again because of the **onlyif**.  The anacron state files (in /var/spool/anacron) start out as a zero byte file until it is run one.  From then on they are updated to a datestamp of when they were last run.   The anacron -u forces the update of the datestamp file without running the commands.

Reply:
thanks for the explanation!